### DISCUSSION TITLE
Clear cache

### DISCUSSION THREAD

**Discussion by teocto** [mixcore/mix.core#684] [Q&A]
![image](https://github.com/mixcore/mix.core/assets/43596506/afc1442d-900b-4185-be9e-8bef0bc07999)
![image](https://github.com/mixcore/mix.core/assets/43596506/c5d104c0-be37-4939-b5bd-24e4dd0be4d1)
I'm changing href value of a, but it doesn't change. Tried clear cache but won't work

**Comment by Smilefounder** [comment]
Which is the correct value that you have changed? As from the content body, the value is correct to the front-end.
![image](https://github.com/mixcore/mix.core/assets/3785721/9cc62406-19e6-4069-a0a7-068a30bb0c41)

![image](https://github.com/mixcore/mix.core/assets/3785721/7d8d03bd-2a44-470e-8065-457ec45e5b73)

**Reply by teocto** [reply]
I'm checking locally and it has to be just '/contract/'. I've solved this by creating new page and replacing it.
